src/redux/version/versionSlice.ts

```diff
@@ -0,0 +1,21 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+import { OpenSpaceVersion } from '@/types/types';
+
+import { getVersion } from './versionMiddleware';
+
+const initialState: OpenSpaceVersion = {};
```

Comment:
Hm, for the other states we have the type for the state in the slice. Feel like maybe we should do that here too? 

It makes it much easier to see what information is part of the state

Reply:
To clarify, based on the comment below. I would prefer if here we had it like this: 

```
interface OpenSpaceVersionState {
  latestOpenSpaceVersion?: SematicVersion;
  openSpaceVersion?: SematicVersion;
  socketApiVersion?: SematicVersion;
}
 
const initialState: OpenSpaceVersionState = {};
```

To make it more consistent with the other states we have.  And these three version numbers are not really used anywhere else